plugin/federation/federation.go

```diff
@@ -243,100 +246,102 @@ func (f *federation) setEntities(schema *ast.Schema) {
 		}
 		if schemaType.Kind == ast.Object {
 			keys := schemaType.Directives.ForNames("key")
-			if len(keys) > 1 {
-				// TODO: support multiple keys -- multiple resolvers per Entity
-				panic("only one @key directive currently supported")
+			if len(keys) == 0 {
```

Comment:
Food for thought perhaps for later PRs.  One of the things that makes this code kinda hard to grok unless you are very familiar with federation internals is the lack of human readable conditionals.  Like I'd love to have a helper function here that's perhaps called something is `isFederatedEntity` or something.  And that checks `schemeType.Kind == ast.Object` and ensure that there are federated keys.

But yeah not for this PR.  Just thoughts I have had about how to make this code more readable and accessible to folks.

Reply:
Went ahead and added this. Hopefully returning the keys isn't more confusing.

Reply:
Yeah I think its great.  The only nit is that functions that start with `is` usually are just predicate functions.  I would have called this instead perhaps `getFederatedEntityKeys` or something and that can return they keys and ok...

but this is perfect.  We can always iterate on this nits.